feat: custom operation resolver

bluebrown opened the pull request:
Signed-off-by: Nico Braun <rainbowstack@gmail.com>

resolves: #360

seriousme commented:
Could you please add a test to https://github.com/seriousme/fastify-openapi-glue/blob/master/test/test-plugin.v3.js ?

bluebrown commented:
I have fixed the tests and added some more tests. I needed to change the invalid service to a string since when it's not given at all now and no service is given, a different error is thrown than before.

seriousme commented:
Thx !
